.lycheeignore

```diff
@@ -1,3 +1,7 @@
 http://localhost:
 https://github.com/edgelesssys/wiki/blob/master/documentation/constellation/customer-onboarding.md
 https://github.com/edgelesssys/wiki/blob/master/documentation/rebasing_forks.md
+# medium.com based links disabled for now until Lychee gains cookie support (medium.com requires cookies to avoid infinite redirects)
```

Comment:
Maybe add tracking link to the GitHub issue?
Otherwise lgtm

Reply:
Can add the tracking link.
The KVM website is a bit funky since the certificate is valid, but they don't seem to serve the intermediate certificate in the chain. Your browser may automatically fetch it.

In the terminal, this fails on Ubuntu 22.04 & Fedora 38 even with the most recent ca-certificates:
```
curl -v https://www.linux-kvm.org/page/Main_Page
```

On macOS, this works just fine in contrast for example.